README.md

```diff
@@ -24,3 +24,52 @@ Apache header:
     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
     See the License for the specific language governing permissions and
     limitations under the License.
+
+
+# CUJ recording
+
+## Usage
+
+### executeCuj
+
+To get data on the latency of a CUJ running on a connected device, run 
+```bash
+./executeCuj <preparatory_actions> <measured_actions> <num_iterations> <r?>
+```
+where preparatory_actions consists of actions to be performed prior to the CUJ 
+in question and measured actions consists of the CUJ in question plus a final 
+"termination" action, which will only be able to be performed once the last
+action in the CUJ has fully terminated. num_iterations denotes the number of 
+iterations you'd like the test to run through your CUJ. Optionally, you may 
+follow the command with "r" to signify that you'd like the run of median length
+to be outputted by the program. (Note that if you'd like to record, the entire
+test, including eachiteration, much have length <= 3 minutes.)
+
+the action list arguments must be formatted as follows:
+
+```bash
+'{<action1>, <action2>, ... , <actionN>}'
```

Comment:
The action themselves may contain commas.  It isn't too big of an issue. But if you take in a file as input, then this problem goes away.